src/GeneralStateTestsFiller/stPreCompiledContracts2/CALLBlake2fFiller.json

```diff
@@ -0,0 +1,418 @@
+{
+  "CALLBlake2f": {
+    "env": {
+      "currentCoinbase": "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+      "currentDifficulty": "0x20000",
+      "currentGasLimit": "10000000000",
+      "currentNumber": "1",
+      "currentTimestamp": "1000",
+      "previousHash": "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+    },
+    "expect": [
+      {
+        "indexes": {
+          "data": [0],
+          "gas": -1,
+          "value": -1
+        },
+        "network": [">=Istanbul"],
+        "result": {
+          "1000000000000000000000000000000000000000": {
+            "storage": {
+              "0x01": "0x01",
```

Comment:
a comment about why expect storage values are expected to be like this

Reply:
covered in "// data" section.